### DISCUSSION TITLE
Regarding obtaining output content from a terminal.

### DISCUSSION THREAD

**Discussion by Carson-yy** [microsoft/vscode-discussions#607] [Extension Development QnA] [answered]
May I ask how could I get the output of a terminal created by vscode.window.createTerminal or an active terminal?

**Comment by r3m0t** [comment, marked as answer]
https://github.com/microsoft/vscode-discussions/discussions/395

**Comment by olguzzar** [comment]
Closing this discussion as it is a duplicate of the discussion linked in the marked answer.
